core-feature-pack/common/src/main/resources/content/bin/systemd/README

```diff
@@ -0,0 +1,80 @@
+= How to configure WildFly as a Systemd service
+
+The following steps describe the process to configure WildFly as a Systemd service.
+You can use the provided wildfly-[standalone,domain].service file as a template for your own Systemd Unit File.
+It can be adjusted to your needs, for example, you can manually change the user that runs the service,
+the location of the WildFly installation, logs, etc.
+Alternatively, you can use the generate_systemd_unit.sh script to automatically generate a new Systemd Unit File
+using this server installation as the WildFly home.
+
+If you want to install WildFly as a Systemd service from scratch as a Systemd service, follow the steps below.
+
+== Install WildFly
+
+    # unzip -d /opt wildfly-34.0.0.Final.zip
+    # ln -s /opt/wildfly-34.0.0.Final /opt/wildfly
+
+== Standalone or Domain mode ?
+
+   # MODE=standalone
+
+== Create a wildfly user and group
+
+    # groupadd -r wildfly
+    # useradd -r -g wildfly -d /opt/wildfly -s /sbin/nologin wildfly
+
+== Configure systemd
+
+    # cd /opt/wildfly/bin/systemd
+    # cp wildfly-$MODE.conf /etc/sysconfig/
+    # cp wildfly-$MODE.service $(pkg-config systemd --variable=systemdsystemunitdir)
+    # chown -R wildfly:wildfly /opt/wildfly/
+    # chmod +x /opt/wildfly/bin/systemd/launch.sh
+    # systemctl daemon-reload
+
+=== If SELinux is enabled, you need to set the appropriate context for the launch.sh script:
+
+    # sudo chcon -R -t bin_t /opt/wildfly/
+    # sudo semanage fcontext -a -t systemd_unit_file_t '/opt/wildfly(/.*)?'
+
+== Start and enable WildFly
+
+    # systemctl start wildfly-$MODE.service
```

Comment:
To respect Bash's best practises:
`# systemctl start "wildfly-${MODE}.service"`

Reply:
Done